docker/install_chrome_dependencies.py

```diff
@@ -27,7 +27,7 @@ def ldd(file_path):
     # For simplicity, I'm assuming if we get an error, the code is non-zero.
     try:
         result = subprocess.run(
-            ["ldd", file_path], capture_output=True, text=True,
+            ["ldd", file_path], capture_output=True, text=True, check=False,
```

Comment:
Since `subprocess.CalledProcessError` appears to only be raised when `check=True`, we should do that here.

> If check is true, and the process exits with a non-zero exit code, a [CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured. ([ref](https://docs.python.org/3/library/subprocess.html#subprocess.run))

I guess we should leave `check=False` on the other line since the exception would be unhandled? I'm not really sure why we're running `ldd` twice here in the first place